wikimedia_publish.py

```
@@ -2,639 +2,206 @@
-            formatted_version = f"{version_info[:4]}-{version_info[4:6]}-{version_info[6:8]}"
-    except:
-        formatted_version = version_info
-    
-    files = job_data.get('files', {})
-    
-    # Group files by content variant to create separate distributions
+        async with session.get(url) as response:
+            response.raise_for_status()
+            async for chunk in response.content.iter_chunked(1024 * 1024):
+                sha256.update(chunk)
+    except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)